src/DynamoAnalyzer/Analyzer/IAnalyze.cs

```diff
@@ -0,0 +1,13 @@
+using DynamoAnalyzer.Models;
+
+namespace DynamoAnalyzer.Analyzer
+{
+    /// <summary>
+    /// Allows to process a package
+    /// </summary>
+    public interface IAnalyze
+    {
+        string Name { get; }
+        Task<AnalyzedPackage> Process();
+    }
+}
```

Comment:
what the the Assets folder and .gitkeep file for?

Reply:
This folder is meant to hold a csv file with dll names we should skip to analyze, because are microsoft dll dependencies or can be found in nuget packages, this is a work in progress, currently, the application generates a file with dll names that were found in some packages, we need to validate the names